CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ We accept contributions via Pull Requests on [Github](https://github.com/:vendor
 
 ## Pull Requests
 
-- **[PSR-2 Coding Standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)** - Check the code style with ``$ composer check-style`` and fix it with ``$ composer fix-style``.
+- **[PSR-12 Coding Standard](https://www.php-fig.org/psr/psr-12/)** - Check the code style with ``$ composer check-style`` and fix it with ``$ composer fix-style``.
```

Comment:
Due to the changes happening to the dev community (master/main branches) I updated this to point to the website instead.

(friendly ping again)